src/angular-google-analytics.js

```diff
@@ -167,6 +167,18 @@ angular.module('angular-google-analytics', [])
         created = true;
       }
 
+      function _generateCommandName(commandName, config) {
+        if (!angular.isUndefined(config) && 'name' in config && config.name) {
+          return config.name+'.'+commandName;
```

Comment:
Nit: give some breathing room to the parts of this statement. Everywhere else string concatenation uses spacing so we should keep that consistent for reading.